cg/server/api.py

```diff
@@ -216,11 +222,25 @@ def parse_case(case_id):
     return jsonify(**case.to_dict(links=True, analyses=True))
 
 
+@BLUEPRINT.route("/cases/delivery_message", methods=["GET"])
+def get_cases_delivery_message():
+    delivery_message_request: DeliveryMessageRequest = DeliveryMessageRequest.model_validate(
+        request.args
+    )
+    service = DeliveryMessageService(db)
```

Comment:
If possible, instantiate the delivery message service like the order service in the `ext` module and import it instead

Reply:
Will give it a try!

Reply:
Moved it!